src/main/java/com/minecolonies/coremod/entity/ai/citizen/enchanter/EntityAIWorkEnchanter.java

```diff
@@ -136,7 +136,7 @@ private IAIState decide()
                 final int numberOfBooksInBuilding = InventoryUtils.getItemCountInProvider(getOwnBuilding(), IS_BOOK);
                 if (numberOfBooksInBuilding > 0)
                 {
-                    needsCurrently = IS_BOOK;
+                    needsCurrently = new com.minecolonies.api.util.Tuple<>(IS_BOOK,1);
```

Comment:
needs to be like this because the minecraft Tuple is used also. can not include both.

Reply:
Then please switch out the other tuple for minecraft tuple

Reply:
if i change 139 to new Tuple<> it errors out with can not infere arguments.. if i change the import also at the top Line 237 errors with incompatible values. the com.minecolonies.api.util. was putted by intellij automaticly

Reply:
Modified to the minecraft version.